File: src/Patterns/RecursivePattern/Python/TriangularNumberPattern.py
def main():
    n = int(input())
    p=n
    triangularNumberPattern(n,p)

def triangularNumberPattern(n,p):
    if(n==0):
        exit()
    print(str(p-n+1)*(p-n+1))
    triangularNumberPattern(n-1,p)

main()

File: src/Patterns/RecursivePattern/TriangularNumberPattern.java
package Patterns.RecursivePattern;

public class TriangularNumberPattern extends Patterns{
    public void pattern(int n, int p){
        if(p>n){
            return;
        }
        for(int i = 1 ; i<=p; i++){
            System.out.print(p);
        }
        System.out.println();
        pattern(n,p+1);
    }

}
